README.md

```diff
@@ -1 +1,35 @@
-# node-debugger-lab
\ No newline at end of file
+# Debug a Script Lab
+
+## Objectives
+
+1. Use Node built-in debugger by putting statements into a given script to find a problem by using debugger
+2. Launch debugger
+3. Fix the script
+
+
+
+## Introduction
+
+In this lab, you'll debug a Node script which has an issue.
+
+## Instructions
+
+1. Open `account.js` then launch it from a command line with `$ node account`.
+2. Notice some issues (there a few of them)
+3. Put some `debugger` statements in `account.js`
```

Comment:
are there issues in every function? maybe give some direction as to where to start with the debugger